media/js/firefox/australis/common.js

```diff
@@ -68,10 +43,26 @@
 
     // link directly to Firefox Accounts when clicking the Sync CTA button
     Mozilla.UITour.getConfiguration('sync', function (config) {
-        $('.sync-cta').on('click', '.button', trackSyncClick);
+        $('.sync-cta .button').each(function() {
+            $(this).attr({
+                'data-page-name': pageId,
+                'data-interaction': 'button click',
+                'data-element-location': 'Get Started with Sync',
+            }).on('click', Mozilla.UITour.showFirefoxAccounts);
```

Comment:
Need to re-instate the Firefox version check (`if (window.getFirefoxMasterVersion() >= 31)`) in this event handler before calling `Mozilla.UITour.showFirefoxAccounts`.